src/newgrf.cpp

```diff
@@ -6250,9 +6250,17 @@ static void GraphicsNew(ByteReader *buf)
 		if (offset <= depot_no_track_offset && offset + num > depot_no_track_offset) _loaded_newgrf_features.tram = TRAMWAY_REPLACE_DEPOT_NO_TRACK;
 	}
 
+	/* If the baseset or grf only provides sprites for flat tiles (pre #10282), duplicate those for use on slopes. */
+	bool dup_oneway_sprites = ((type == 0x09) && (offset + num) <= SPR_ONEWAY_SLOPE_N_OFFSET);
```

Comment:
Parenthesis don't make sense here: the `)` after `num` should be after the offset constant.

Reply:
Fixed now, thanks.